api/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to web-client for Reach4Help
+
+🎉🚀🙌🏻 First off, thanks for taking the time to contribute! 🙌🏻🚀🎉
+
+> First off, thank you for considering contributing to this repository. It's people like you that make Open Source so great.
+
+### Please read our guidelines before you start.
+
+> Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### Contributions we are looking for.
+
+We keep an open mind! Improving documentation, bug triaging, or writing tutorials are all examples of helpful contributions that mean less work for you.
+
+# Ground Rules
+
+### Follow our Code of Conduct.
+
+We expect everyone to abide by our [**Code of Conduct**](CODE_OF_CONDUCT.md). Please read it. 🤝
+
+# Your First Contribution
+
+Here are a couple of friendly tutorials to help you get started: http://makeapullrequest.com/ and http://www.firsttimersonly.com/
+
+> Working on your first Pull Request? You can learn how from this _free_ series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first :smile_cat:
+
+# Getting started
+
+## Prerequisites
+
+### You will need at least `node` and `yarn` installed.
+
+- [Download Node Here](https://nodejs.org/en/download/ 'Download Node Here')
+- [Download Yarn Here](https://yarnpkg.com/lang/en/docs/install/ 'Download Yarn Here')
+
+#### In this repo we favour `yarn` over `npm` as the "official" package manager since we also leverage `yarn workspaces` as the manager for our `monorepo`
+
+## Run the project
+
+1 - Install dependencies
+
+```
+yarn install
+```
+
+2 - Setup database and adminer
+
+```
+docker-compose up
+```
+
+3 - Create a `.env` file in the project root using the `example.env`.
+
+4 - Run the `api` project in development mode.
+
+```
+yarn run start:dev
+```
+
+5 - Run database migrations
+
+```
+yarn typeorm migration:run
+```
+
+6 - The api swagger-ui is now available under [localhost:3001](http://localhost:3001)
+
+
+# How to report a bug
+
+> If you find a security vulnerability, open an issue describing the problem.
```

Comment:
We don't want people opening issues to report security vulnerabilities, we want them to disclose ethically. @mdeous will be writing up and publishing our security policy soon, this readme should refer to that when available, until then, change this to the following:

```suggestion
> If you find a security vulnerability, please send a report to <security@reach4help.org>
```